src/behavior-considered-undefined.md

```diff
@@ -71,12 +71,17 @@ Please read the [Rustonomicon] before writing unsafe code.
   * A `!` (all values are invalid for this type).
   * An integer (`i*`/`u*`), floating point value (`f*`), or raw pointer obtained
     from [uninitialized memory][undef], or uninitialized memory in a `str`.
-  * A reference or `Box<T>` that is [dangling], misaligned, or points to an invalid value.
-  * Invalid metadata in a wide reference, `Box<T>`, or raw pointer:
+  * A reference or `Box<T>` that is [dangling], misaligned, or points to an invalid value
+    (using the actual dynamic type of the pointee as determined by the vtable in
+    the metadata in case of dynamically sized types).
+  * Invalid metadata in a wide reference, `Box<T>`, or raw pointer. The requirement
+    for the metadata is determined by the type of the unsized tail:
```

Comment:
Another piece of clarification, previously we were not very clear on what the metadata requirements are for types like `(u32, [u32])`: we spoke about "slice metadata" and the fact that this covered all types whose unsized tail is a slice was left implicit. Now it is explicit.